templates/data/incubator-buildpack-data.yaml

```diff
@@ -230,6 +237,26 @@ templates:
               GARDENER_ZONES: "europe-west4-b"
             annotations:
               pipeline.platform: gardener_gcp
+          reconciler_e2e_upgrade_jobConfig:
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/reconciler-e2e-upgrade-gardener.sh"
+            args:
+              - "/home/prow/go/src/github.com/kyma-incubator/reconciler"
+            branches:
+              - ^main$
+          reconciler_gardener_e2e_upgrade:
+            labels:
+              preset-gardener-gcp-kyma-integration: "true"
+              preset-kyma-cli-stable: "true"
+              preset-cluster-version: "true"
+            env:
+              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
+              GARDENER_REGION: "europe-west4"
+              GARDENER_ZONES: "europe-west4-b"
+            annotations:
```

Comment:
lets use [gardener_gcp_job](https://github.com/kyma-project/test-infra/blob/main/templates/data/kyma-integration-gardener-data.yaml#L67) and avoid hard coding here